code/game/objects/items/weapons/tools/hammer.dm

```diff
@@ -152,3 +152,31 @@
 		playsound(src, 'sound/machines/hiss.ogg', 50, 0, 0)
 		user.throw_at(target, get_dist(target, user), 1, user)
 		T.stop()
+
+/obj/item/weapon/tool/hammer/ironhammer
+	name = "FS \"Ironhammer\" Breaching Hammer"
+	desc = "A modified sledgehammer produced by Frozen Star for Ironhammer forces. This tool can take down standard walls in about eight swings, and can take down reinforced walls in about fifty swings. However, this tool requires significant strength to use efficiently."
+	icon_state = "ironhammer"
+	item_state = "ironhammer"
+	wielded_icon = "ironhammer_doble"
+	force_unwielded = WEAPON_FORCE_ROBUST
+	force_wielded = WEAPON_FORCE_BRUTAL
+	armor_penetration = ARMOR_PEN_EXTREME
+	structure_damage_factor = STRUCTURE_DAMAGE_DESTRUCTIVE
+	w_class = ITEM_SIZE_HUGE
+	slot_flags = SLOT_BACK
+	origin_tech = list(TECH_ENGINEERING = 3, TECH_COMBAT = 1, TECH_MATERIAL = 1)
+	tool_qualities = list(QUALITY_HAMMERING = 40, QUALITY_PRYING = 20)
+	matter = list(MATERIAL_PLASTEEL = 4, MATERIAL_STEEL = 2, MATERIAL_PLASTIC = 2)
+
+/obj/item/weapon/tool/hammer/ironhammer/wield(mob/living/user)
+	..()
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		var/rob_stat = H.stats.getStat(STAT_ROB)
```

Comment:
IH shouldnt be expected to do maint looting since they have a job that requires them to always be on standby, though their starting 25 ROB is too low since most IH get origins which increase their ROB by at least 5 so 30 is fair

Reply:
>It shouldn't be a flat 'you can't use this if you have less than 20 ROB' - instead, make people with under the required amount take far longer to take down walls. Double, maybe triple the time.

actually what genesse said is a lot better though the ROB requirement should be higher since origins exist imo